static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -45,24 +47,12 @@ export function rrwebEventListFactory(
   return events;
 }
 
-function entriesFromEvents(events: Event[]) {
-  return events.flatMap(event => event.entries);
-}
-
-export function breadcrumbValuesFromEvents(events: Event[]) {
-  const fromEntries = entriesFromEvents(events).flatMap(entry =>
-    entry.type === EntryType.BREADCRUMBS ? entry.data.values : []
-  );
-  const fromEvents = events.map(getVirtualCrumb).filter(Boolean) as RawCrumb[];
-
-  return ([] as RawCrumb[]).concat(fromEntries).concat(fromEvents);
-}
```

Comment:
We now send `RawCrumbs` with our replay attachments, no need for events and grabbing spans/breadcrumbs from events.